### ISSUE TITLE
do_tensorflow.py assumes classification

### ISSUE THREAD

**Issue by vatai** [undertherain/benchmarker#82]
In `do_tensorflow.py` [line 77](https://github.com/undertherain/benchmarker/blob/baa5e7fe950ecd6cd8bd91b685b27d4057394ea9/benchmarker/modules/do_tensorflow.py#L77), we apply `to_categorical` to `y_train`. This is needed for classification tasks but messes up e.g. MSE loss.

**Comment by vatai** [comment]
I think I have a solution. The `get_kernel` for classification tasks should be modified to use [SPARSE categorical cross-entropy](https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy) instead of (simple, plane) [categorical cross-entropy)[https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy]
